expat/xmlwf/xmlwf.c

```diff
@@ -322,6 +322,14 @@ freeNotations(XmlwfUserData *data) {
   data->notationListHead = NULL;
 }
 
+static void
+cleanupUserData(XmlwfUserData *userData) {
+  if (userData->currentDoctypeName != NULL) {
```

Comment:
Do we need to check for `NULL`here? `man 3 free`tells me that the `free` function can handle `NULL` fine.